WordPress/Classes/Stores/PluginStore.swift

```diff
@@ -334,6 +403,37 @@ private extension PluginStore {
         })
     }
 
+    func installPlugin(slug: String, site: JetpackSiteRef) {
+        guard let remote = remote(site: site),
+            !state.updatesInProgress[site, default: Set()].contains(slug),
+            getPlugin(slug: slug, site: site) == nil else {
+                return
+        }
+
+        transaction { state in
+            state.updatesInProgress[site, default: Set()].insert(slug)
```

Comment:
I don't think there needs to be a transaction if there's only one change to state. `transaction` only groups changes so that there's a single `onChange` event, but it shouldn't make any difference here